Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1

```diff
@@ -5,9 +5,13 @@ param (
     [string]$targetPath,
     [object]$credential,
     [string]$cleanTargetBeforeCopy,
-    [string]$additionalArguments
+    [string]$additionalArguments,
+    [string]$scriptRoot
     )
 
+    Import-Module "$scriptRoot\ps_modules\VstsTaskSdk" 
+    Import-VstsLocStrings -LiteralPath $scriptRoot/Task.json
```

Comment:
type of slash differs in two import statement. Can you verify if the localized string is correctly interpreted?

Reply:
I have checked by adding wrong source path . Message is : Source path 'C:\agent\_work\r1\a\$' does not exist. It is correctly interpreted. I have copied from other file it has also used same slash for each import .